openassessment/templates/openassessmentblock/staff_debug/student_info.html

```diff
@@ -25,12 +25,11 @@ <h3 class="title">{% trans "Student Response" %}</h3>
                     {% endif %}
                 </div>
 
-                {% if submission.image_url %}
```

Comment:
Following up on this, I now see that this was removed, but I'm ok with the change, and am not necessarily suggesting we do anything different here. Thanks!